packages/website/docusaurus.config.js

```diff
@@ -108,13 +108,17 @@ const config = {
             position: "left",
             label: "API",
           },
-          { to: "/blog", label: "Blog", position: "left" },
           { to: "/playground", label: "Playground", position: "left" },
           {
             href: "https://github.com/quantified-uncertainty/squiggle/discussions",
             label: "Issues & Discussion",
             position: "right",
           },
+          {
+            href: "https://quri.substack.com/t/squiggle",
+            label: "Newsletter",
+            position: "right",
+          },
```

Comment:
You've imported the 3 old blog posts there, great. That makes my nextra migration task significantly easier.